src/SnmpLibrary/library.py

```diff
@@ -390,7 +395,8 @@ def walk(self, oid):
             self._active_connection.cmd_gen.nextCmd(
                 self._active_connection.authentication_data,
                 self._active_connection.transport_target,
-                oid
+                oid,
+                contextName=self._active_connection.context_name,
```

Comment:
please remove the ',' from the end of the line

Reply:
Done in 0f18d150a6ac21aaf7dcf4dfddbf732bcacc3ace